tensorflow/lite/delegates/webnn/resize_bilinear_test.cc

```diff
@@ -50,7 +50,8 @@ TEST(ResizeBilinear, AlignCenters) {
       .Test(webnn_delegate.get());
 }
 
-TEST(ResizeBilinear, AlignCentersTF1X) {
+//Webnn does not support this option
+TEST(ResizeBilinear, DISABLED_AlignCentersTF1X) {
```

Comment:
Please add link about this corresponding issue: https://github.com/webmachinelearning/webnn/issues/270

Reply:
Please add space after `//`, `#`